auth-web/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "auth-web",
-  "version": "2.0.4",
+  "version": "2.1.0",
```

Comment:
Do we really need to make this a major change? None of the changes really impact auth-web itself

Either way it's fine

Reply:
Severin suggested it in another PR.